CommonClient.py

```diff
@@ -24,7 +24,7 @@
 from MultiServer import CommandProcessor, mark_raw
 from NetUtils import (Endpoint, decode, NetworkItem, encode, JSONtoTextParser, ClientStatus, Permission, NetworkSlot,
                       RawJSONtoTextParser, add_json_text, add_json_location, add_json_item, JSONTypes, HintStatus, SlotType)
-from Utils import Version, stream_input, async_start
+from Utils import gui_enabled, Version, stream_input, async_start
```

Comment:
I think your changes are sensible, but I'm a bit confused about this original Treble stuff.

Imo, there is one of two possibilities:
1. This should be `from Utils import gui_enabled as gui_enabled`
2. Clients are "meant" to switch over to `from Utils import gui_enabled`

As relying on implicit reexporting is bad™️

I would like us to be clear about what the intent is here